.travis.yml

```diff
@@ -1,6 +1,19 @@
 language: rust
 rust:
     - nightly
+
+before_script:
+    - "sh -e /etc/init.d/xvfb start"
+    - "export DISPLAY=:99.0"
+    - "wget http://selenium-release.storage.googleapis.com/2.50/selenium-server-standalone-2.50.1.jar"
```

Comment:
Tried to get this working and it works on my machine, but not on Travis, not sure why. Will make switching to selenium-standalone a follow-up ticket.